Use Cases/migrate-normal-table-to-timescale-table.md

```diff
@@ -0,0 +1,144 @@
+---
+title: "Migrate normal table to timescale table"
+date: 2025-01-15
+tags:
+  - postgresql
+  - timescale
+  - database
+  - hedge foundation
+description: "How do we migrate normal table to timescale table to optimized data storage"
+authors:
+  - minhth
+---
+As high trading volume the data in 2 tables `user_trades` and `incomes` are bigger so it will make our report is slower every day. With this problem, we consider to move these tables to timescale tables to split into multiple chunks by time so it will optimized our query report
+
+## We create new timescale tables
+Because current tables have large data so we can't migrate it in one SQL Query so that why we have create new table with new timescale structure and this is new table structures
+
+### New ts_user_trades
+```sql
+CREATE TABLE "public"."ts_user_trades" (
+    "id" uuid NOT NULL DEFAULT uuid_generate_v4(),
+    "buyer" bool,
+    "commission" numeric,
+    "commission_asset" text,
+    "trade_id" int8 NOT NULL,
+    "maker" bool,
+    "order_id" int8,
+    "price" numeric,
+    "qty" numeric,
+    "quote_qty" numeric,
+    "realized_pnl" numeric,
+    "side" text,
+    "position_side" text,
+    "symbol" text,
+    "time" timestamptz NOT NULL,
+    "time_unix" int8,
+    "account_id" uuid,
+    "is_locked_position" bool DEFAULT false,
+    "created_at" timestamptz NOT NULL DEFAULT now(),
+    "updated_at" timestamptz NOT NULL DEFAULT now(),
+    CONSTRAINT "ts_user_trades_account_id_fkey" FOREIGN KEY ("account_id") REFERENCES "public"."accounts"("id"),
+    PRIMARY KEY ("id","time")
+);
+
+-- Indices
+CREATE UNIQUE INDEX ts_user_trades_account_id_symbol_trade_id_time_index ON public.ts_user_trades USING btree (account_id, symbol, trade_id, "time");
+CREATE INDEX ts_user_trades_account_id_time_index ON public.ts_user_trades USING btree (account_id, "time");
+CREATE INDEX ts_user_trades_symbol_index ON public.ts_user_trades USING btree (symbol);
+CREATE INDEX ts_user_trades_trade_id_index ON public.ts_user_trades USING btree (trade_id);
+CREATE INDEX ts_user_trades_time_unix_index ON public.ts_user_trades USING btree (time_unix);
+CREATE INDEX ts_user_trades_time_idx ON public.ts_user_trades USING btree ("time" DESC);
+CREATE INDEX ts_user_trades_account_id_symbol_time_index ON public.ts_user_trades USING btree (account_id, symbol, "time");
+
+SELECT create_hypertable('ts_user_trades', 'time',
+  chunk_time_interval => INTERVAL '1 day',
+  if_not_exists => TRUE
+);
```

Comment:
Which parts are introduced to new table? index using btree? hypertable? why we need to add them?